docs/csharp/getting-started/with-visual-studio-code.md

```diff
@@ -15,14 +15,43 @@ ms.assetid: 76c23597-4cf9-467e-8a47-0c3703ce37e7
 
 # 🔧 Getting started with Visual Studio Code
 
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/944) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
+.NET Core gives you a blazing fast and modular platform for creating server applications that run on Windows, Linux and macOS. Use Visual Studio Code with the C# extension to get a powerful editing experience with full support for C# IntelliSense (smart code completion) and debugging.
+
+## Getting Started
+
+1. Install [.NET Core](https://microsoft.com/net/core).
+2. Install the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp) from the VS Code Marketplace.
```

Comment:
Since this is not part of VS Code docs anymore, should the list include installing VS Code itself?